packages/pages/src/dev/server/ssr/generateTestData.ts

```diff
@@ -60,6 +60,23 @@ export const generateTestDataForSlug = async (
   const args = getCommonArgs(featuresConfigForEntityPages, projectStructure);
   args.push("--slug", slug);
 
+  const slugFields = new Set<string>();
+  let shouldAddDefaultSlugField = false;
+  templateModuleCollection.forEach((templateModule) => {
+    const slugField = templateModule?.config?.slugField;
+    if (slugField) {
+      slugFields.add(`entity.${slugField}`);
+    } else {
+      shouldAddDefaultSlugField = true;
```

Comment:
This else could just add entity.slug and then you can can rid of lines 73-78 (and always do the args.push now).